src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -20,12 +23,12 @@
 class BreadcrumbsHelper extends \Cake\View\Helper\BreadcrumbsHelper
 {
     /**
-     * Default config for the helper.
+     * Default configuration for this helper.
+     * Don't override parent::$_defaultConfig for robustness
      *
      * @var array
-     * @link https://api.cakephp.org/3.3/class-Cake.View.Helper.BreadcrumbsHelper.html
      */
-    protected $_defaultConfig = [
+    protected $helperConfig = [
```

Comment:
Cake convention is `_`-prefix for `protected` members so maybe use `$_helperConfig`? (Also applies to other changes.)